app/org/apache/spark/deploy/history/SparkFSFetcher.scala

```diff
@@ -338,6 +339,7 @@ private object SparkFSFetcher {
 
   var defEventLogDir = "/system/spark-history"
   var defEventLogSizeInMb = 100d; // 100MB
+  var defSparkLogExt = "_1.snappy"
```

Comment:
Actually `_1` means 1st try. There will be log files ended with `_N` indicating it's the Nth try. We previously discard all further tries just for simplicity. But generally this is not a very correct behavior. Maybe further improvements could be trying to fetch the latest try instead of the 1st try?